Readme.md

```diff
@@ -40,23 +40,18 @@ $ npm install -g pino-pretty
 + `--colorize` (`-c`): Adds terminal color escape sequences to the output.
 + `--crlf` (`-f`): Appends carriage return and line feed, instead of just a line
 feed, to the formatted log line.
-+ `--dateFormat` (`-d`): Sets the format string to apply when translating the date
-to human readable format (see: `--translateTime`). The default format string
-is `'yyyy-mm-dd HH:MM:ss.l o'`. For a list of available patter letters
-see the [`dateformat` documentation](https://www.npmjs.com/package/dateformat).
-When the value is anything other than the default value, `--translateTime` is
-implied.
 + `--errorProps` (`-e`): When formatting an error object, display this list
 of properties. The list should be a comma separated list of properties Default: `''`.
 + `--levelFirst` (`-l`): Display the log level name before the logged date and time.
 + `--errorLikeObjectKeys` (`-k`): Define the log keys that are associated with
 error like objects. Default: `err,error`.
 + `--messageKey` (`-m`): Define the key that contains the main log message.
 Default: `msg`.
-+ `--localTime` (`-n`): When translating the time to a human readable format,
-use the system timezone for displaying the time.
 + `--translateTime` (`-t`): Translate the epoch time value into a human readable
-date and time string. See `--dateFormat` for information on the output format.
+in `UTC`. The default pattern is `'yyyy-mm-dd HH:MM:ss.l o'` time in UTC.
+IF want to translate to the local system's timezone, require `SYS:` prefix at 
```

Comment:
Replace this sentence with: "If you want to translate local system's timezone then you must prefix the format string with `SYS:`, e.g. `SYS:yyyy-mm-dd HH:MM:ss`."

Reply:
this ?

+ `--translateTime` (`-t`): Translate the epoch time value into a human readable
date and time string in `UTC`. The default pattern is `'yyyy-mm-dd HH:MM:ss.l o'`.
IF you want to translate local system's timezone, then you must prefix the format 
string with `SYS:`, e.g. `'SYS:yyyy-mm-dd HH:MM:ss'`. For a list of available patter
letters see the [`dateformat` documentation](https://www.npmjs.com/package/dateformat#mask-options).

```
+ `--translateTime` (`-t`): Translate the epoch time value into a human readable
date and time string in `UTC`. The default pattern is `'yyyy-mm-dd HH:MM:ss.l o'`.
IF you want to translate local system's timezone, then you must prefix the format 
string with `SYS:`, e.g. `'SYS:yyyy-mm-dd HH:MM:ss'`. For a list of available patter
letters see the [`dateformat` documentation](https://www.npmjs.com/package/dateformat#mask-options).
```

Reply:
Yes, but fix "IF" in the third sentence to "If".

Reply:
sorry, which one?

+ `--translateTime` (`-t`): Translate the epoch time value into a human readable
date and time string in `UTC`. The default pattern is `'yyyy-mm-dd HH:MM:ss.l o'`.
If you want to translate local system's timezone, then you must prefix the format 
string with `SYS:`, e.g. `'SYS:yyyy-mm-dd HH:MM:ss'`. See [`dateformat` documentation](https://www.npmjs.com/package/dateformat#mask-options)
for more available pattern letters.

+ `--translateTime` (`-t`): Translate the epoch time value into a human readable
date and time string in `UTC`. The default pattern is `'yyyy-mm-dd HH:MM:ss.l o'`.
If you want to translate local system's timezone, then you must prefix the format 
string with `SYS:`, e.g. `'SYS:yyyy-mm-dd HH:MM:ss'`. For a list of available patten
letters see the [`dateformat` documentation](https://www.npmjs.com/package/dateformat#mask-options).